src/cplscheme/impl/ConvergenceMeasure.hpp

```diff
@@ -48,6 +48,12 @@ class ConvergenceMeasure {
   {
     return 0;
   }
+
+  /// Returns an abreviation of the name of the measure for the log file headers
+  virtual std::string getAbbreviation() const
+  {
+    return "";
+  }
```

Comment:
You could also force this to be provided by the convergence measure by making it abstract.
```suggestion
  virtual std::string getAbbreviation() const = 0;
```

Reply:
I copied from `getNormResidual()`. Problem is that both functions make no sense for `MinIterationsConvergenceMeasure`. Should get cleaner with #800

Reply:
Ok, let's refactor both together in the future.